facilitator/src/aggregation.rs

```diff
@@ -42,7 +43,7 @@ impl<'a> BatchAggregator<'a> {
         share_processor_signing_key: &'a EcdsaKeyPair,
         peer_share_processor_key: &'a UnparsedPublicKey<Vec<u8>>,
         share_processor_ecies_key: &'a PrivateKey,
-    ) -> Result<BatchAggregator<'a>, Error> {
+    ) -> anyhow::Result<BatchAggregator<'a>> {
```

Comment:
No, nothing terrible happens. It's just that since the current code already uses `std::Result`, importing both at random across the codebase would be confusing. After everything is converted we can do a pass and just import it everywhere.

Reply:
In retrospect I could have actually imported it, since if you pass both parameters to it, it works exactly the same. (The definition is just `pub type Result<T, E = Error> = core::result::Result<T, E>;`)

Reply:
I said "in retrospect" but I can just put this in the change right now, so I'll do that later.